lib/reverse_string.rb

```diff
@@ -1,5 +1,13 @@
 def reverse_characters(str)
+    array = [str.split("")]
+    reverse_array = []
+    loop.each do
```

Comment:
There are a few ways to do loops in Ruby. This way–using the `loop` keyword–requires a `break` somewhere inside the "block" (That's the code between the `do` and the `end`). Without a `break`, it'll just go on forever.